sql/core/src/test/resources/sql-tests/results/ansi/string-functions.sql.out

```diff
@@ -73,8 +73,8 @@ select left(null, -2), left("abcd", -2), left("abcd", 0), left("abcd", 'a')
 -- !query schema
 struct<>
 -- !query output
-java.lang.NumberFormatException
-invalid input syntax for type numeric: a
```

Comment:
can we move `left(null, -2)` out to a new query? otherwise we can't test the error message of `left("abcd", 'a')`